io.github.tralph3.Steam_Metadata_Editor.metainfo.xml

```diff
@@ -0,0 +1,368 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>io.github.tralph3.steam_metadata_editor</id>
+
+  <name>Steam Metadata Editor</name>
+  <summary>An easy to use GUI that edits the metadata of your Steam Apps</summary>
+
+  <metadata_license>FSFAP</metadata_license>
+  <project_license>GPL-3.0-or-later</project_license>
+
+  <description>
+    <p>
+      [![License: GPL v3](https://img.shields.io/badge/License-GPL%20v3-blue.svg?style=flat-square)](https://github.com/tralph3/Steam-Metadata-Editor/blob/master/LICENSE) [![Donate](https://img.shields.io/badge/Donate-PayPal-blue.svg?style=flat-square)](https://paypal.me/tralph3)
```

Comment:
this will no do what you think it does. Markdown is ignored here, and appstream doesn't like URL in descriptions.

Reply:
Oh I see, I just copied the readme file as it is. I'll trim it down to just contain a description then.

Reply:
Just to clarifiy: this is not suppsed to be a verbati copy of the README